[margeen mi mierda ya] Quirks 8.0: skills departamentales, Modificaciones geneticas, Discapacidades, Implantes y Artes marciales + fix skills antagonistas +fix tools use hispano

Evankhell561 opened the pull request:
## What Does This PR Do
quirks 2.0, 
los quirks pueden superar la muerte
quirks 3.0
los quikrs departamentales
quirks 4.0 
quirks genticos
quirks 5.0
add: quirks mimo y payaso, del: quirks mimo y payaso, add: quirks torpeza y gusfobia
quirks 6.0
quirks Discapacidades, Implantes y Artes marciales.
quirks 7.0 
quirks y antagonistas
## Why It's Good For The Game
ya ni sé
 
## Advertencia
Este pr no ha sido testeado pero tengo absoluta confianza en mis habilidades de PROcoder

## Changelog
:cl:
tweak: los Quirks ahora se mantienen aún después de la clonacion
add: quirk tolerancia al alcohol
add: quirk disparo a dos armas.
del: piloto de sec
add: piloto de sec
add: quirk no_tools
add: quirk piloto de mechas
add: quirk piloto de pods 
add: algunos jobs tienen quirks por defecto.
tweak: las armas no letales de los mechas vienen desbloqueadas por defecto.
add: quirk gunsfobia.
add: quirk torpeza.
tweak: el mutadone es menos efectivo.
add: quirk poor_aim.
tweak: la gente sin conocimiento maneja mal los mechas.
tweak: la gente sin conocimiento maneja mal los pods.
tweak: la gente sin conocimeinto tiene un 50% de probabilidad de fallar una operacion.
add: quirk nutriimplant
add: quirk llama azul
add: quirk manco
add: quirk no craft
add: todo lo que no dije anteriormente
/:cl:

Commit:
a ver que sale

Commit:
Update mind.dm

Commit:
code2.0

el futuro es ahora

Silverexperience reviewed (changes requested):
Lo estuve testeado, y cuando sali de la clonadora con el quirk de pacifismo le di una paliza al CMO que me clono.

Commit:
el anterior commit solo era una prueba para el nuevo prefecto este es el commit real

Evankhell561 commented:
> Lo estuve testeado, y cuando sali de la clonadora con el quirk de pacifismo le di una paliza al CMO que me clono.

Cambié el codigo, a ver ahora

Silverexperience reviewed (changes requested):
Lo estuve testeando, y nada mas salir de clonacion mi personaje pacifico le rompio los huesos a la doctora.

Commit:
quirks 3.0

Evankhell561 commented:
> Lo estuve testeando, y nada mas salir de clonacion mi personaje pacifico le rompio los huesos a la doctora.

silver, guapo. ahora si funcionará, tengo una corazonada.

Commit:
linea borrada sin querer

Commit:
hispania traiis

Commit:
putas lineas tio

Evankhell561 commented:
> Lo estuve testeando, y nada mas salir de clonacion mi personaje pacifico le rompio los huesos a la doctora.

dime que ahora sí perduran a la clonacion por favor.

Silverexperience reviewed (changes requested):
Ahora si que se guardan los traits, pero genera runtimes al solo crearse una persona, incluso sin traits.
![imagen](https://user-images.githubusercontent.com/58746682/80973175-52a69280-8e1f-11ea-9e21-fd0cfe3d37aa.png)

Evankhell561 commented:
> Ahora si que se guardan los traits, pero genera runtimes al solo crearse una persona, incluso sin traits.
> ![imagen](https://user-images.githubusercontent.com/58746682/80973175-52a69280-8e1f-11ea-9e21-fd0cfe3d37aa.png)

creo que esto es debido a que si se crea una mente que no haya tenido un cuerpo anteior, le estoy pidiendo al juego que transfiera sus quirks de su anterior cuerpo al nuevo

Evankhell561 commented:
Tambien quisiera saber si puedo dejar nombres y descripciones en español, además de que quiero recibir feedback con respecto a que seria adecuado en cada una de ellas.

Silverexperience commented:
> 
> 
> Tambien quisiera saber si puedo dejar nombres y descripciones en español, además de que quiero recibir feedback con respecto a que seria adecuado en cada una de ellas.

No veo mal que esten en español los nombres, aunque si quisieras tenerlo totalmente igual puedes ponerlo en ingles.
El quirk de mimo no lo veo muy bien porque al final le quita parte de lo unico que tiene ese trabajo.
Y las descripciones puedes ser algo meme, pero tampoco deberia ser demasiado o con referencias que no pillaria la mayoria como lo de evan.

Evankhell561 commented:
> El quirk de mimo no lo veo muy bien porque al final le quita parte de lo unico que tiene ese trabajo.

Lo estaba dudando sí pero el payaso es una modificacion genetica, puedes modificar tus genes para ello y no por eso la gente deja de jugar payaso, en cambio el mimo es... unico. además de que existiran habilidades interdepartamentales, un ingeniero podrá tomar el quirk de cirujano y así. A mi forma de verlo solo debemos jugar bien con los puntos.

Commit:
más quirks

quirk cirujano listo
quirk piloto de pods wip
quirk dual shooter wip
quirk mech pilot aun en wip

Commit:
opps

el quirk dual shoot ahora sí que sí en wip

Commit:
putas lineas

Silverexperience commented:
> 
> 
> > El quirk de mimo no lo veo muy bien porque al final le quita parte de lo unico que tiene ese trabajo.
> 
> Lo estaba dudando sí pero el payaso es una modificacion genetica, puedes modificar tus genes para ello y no por eso la gente deja de jugar payaso, en cambio el mimo es... unico. además de que existiran habilidades interdepartamentales, un ingeniero podrá tomar el quirk de cirujano y así. A mi forma de verlo solo debemos jugar bien con los puntos.

Pero la voz del payaso no caracteriza al payaso, las habilidades del mimo son una gran parte de lo que es ser mimo.

Evankhell561 commented:
> > > El quirk de mimo no lo veo muy bien porque al final le quita parte de lo unico que tiene ese trabajo.
> > 
> > 
> > Lo estaba dudando sí pero el payaso es una modificacion genetica, puedes modificar tus genes para ello y no por eso la gente deja de jugar payaso, en cambio el mimo es... unico. además de que existiran habilidades interdepartamentales, un ingeniero podrá tomar el quirk de cirujano y así. A mi forma de verlo solo debemos jugar bien con los puntos.
> 
> Pero la voz del payaso no caracteriza al payaso, las habilidades del mimo son una gran parte de lo que es ser mimo.

uhm pues, la voz del payaso no es lo unico que tiene ser payaso, esto tambien hace que puedas golpearte a ti mismo con armas a mele y caer solo por que sí. but, si así lo prefieres lo retiro. tampoco es que estoy muy ligado a ese quirk.

Silverexperience commented:
> 
> 
> > > > El quirk de mimo no lo veo muy bien porque al final le quita parte de lo unico que tiene ese trabajo.
> > > 
> > > 
> > > Lo estaba dudando sí pero el payaso es una modificacion genetica, puedes modificar tus genes para ello y no por eso la gente deja de jugar payaso, en cambio el mimo es... unico. además de que existiran habilidades interdepartamentales, un ingeniero podrá tomar el quirk de cirujano y así. A mi forma de verlo solo debemos jugar bien con los puntos.
> > 
> > 
> > Pero la voz del payaso no caracteriza al payaso, las habilidades del mimo son una gran parte de lo que es ser mimo.
> 
> uhm pues, la voz del payaso no es lo unico que tiene ser payaso, esto tambien hace que puedas golpearte a ti mismo con armas a mele y caer solo por que sí. but, si así lo prefieres lo retiro. tampoco es que estoy muy ligado a ese quirk.

Sera mejor dejar a los payaso y a los mimos de manera unica, aunque si podrias hacer un quirk de torpe y que haga lo mismo que describes del payaso.

Evankhell561 commented:
> Sera mejor dejar a los payaso y a los mimos de manera unica, aunque si podrias hacer un quirk de torpe y que haga lo mismo que describes del payaso.

Lo haré, just, cualquier genetista puede convertirte en payaso con el gen adecuado asi que ya de por sí el gen payaso no es algo unico but, retiraré esos dos en la proxima version.

Commit:
quirk piloto listo

quirk piloto + brigdoc con mechas
+piloto con quirk piloto y mecanico con quirk mecanico

Silverexperience commented:
> 
> 
> > Sera mejor dejar a los payaso y a los mimos de manera unica, aunque si podrias hacer un quirk de torpe y que haga lo mismo que describes del payaso.
> 
> Lo haré, just, cualquier genetista puede convertirte en payaso con el gen adecuado asi que ya de por sí el gen payaso no es algo unico but, retiraré esos dos en la proxima version.

Que yo sepa solo te da voz de comic sans e igualmente el gen no lo tienes desde el inicio.

Evankhell561 commented:
> Que yo sepa solo te da voz de comic sans e igualmente el gen no lo tienes desde el inicio.

Nel, el gen te da la voz más todo lo demás que hace a un payaso payaso
![image](https://user-images.githubusercontent.com/49106866/81015354-8b993400-8e2c-11ea-8c4a-9e27f5faacda.png)

But, como he dicho, lo quitaré. Solo estaba ahi porque queria más quirks negativos. Y también me gusta el razonamiento que me has dado.

Commit:
MÁS COSAS

DEL QUIRK MIMO
DEL QUIRK PAYASO
ADD GEN TORPEZA
ADD QUIRK DUAL SHOOTER

Commit:
más quirks

add gunsfobia
add mech pilot

Commit:
revertspircen

Commit:
Update limb_reattach.dm

Commit:
más quirks

balance de quirks de habilidades
add quirk estomago fuerte

Commit:
tweak no gun

mueve la confirmacion de gunsfobia a human.dm

Commit:
más cosas

No tener el quirk de piloto de pods ahora te permite conducir el pod un 20% de las vecess
fix quirk no gun
fix quirk estomago fuerte
fix quirk cirujano
sin el quirk cirujano tienes una probabilidad de 50% hacer la cirugia con exito.
sin el quirk del mech pilot ahora puedes conducir el 30% de las veces

Commit:
más quirks

add: trait no craft
add: quirk implante de comida
add: quirk implante la llama azul

Commit:
Update positive.dm

Commit:
QUIRK PRO SHOOTER

EL QUIRK PRO SHOOTER SE CAMBIA POR El poor_aim y pasa a ser un trait negativo.

Commit:
elimina el quirk shooter

el quirk shooter fue eliminado
ahora existe el quirk poor aim que te da mala punteria.

Commit:
puta linea x3

Commit:
Update positive.dm

Commit:
SHOOTES IS BACK

ELIMINA EL POOR AIM
AÑADE EL SHOOTER PERO AHORA solo aplica a armas medianas en adelante
aumenta el costo energetico de los lasers normales
fixea el trait de piloto

Commit:
fix quirk pod piloto again

casi todos los secs saben disparar armas grandes

Commit:
antag quirks and runtimes

Añade procs utiles como el remover todos los quirks.
Y añadir todos los quirks necesarios para un traidor.
soluciona el problema de los runtimes al crear un pj.

Commit:
diooos

fixea los attackby de hispania y los pasa a tool_use

Commit:
fixes

fixea la gunsfobia, fixea descripciones y nombres de quirks

Commit:
Merge remote-tracking branch 'upstream/master' into traitsroundstart

Commit:
Merge remote-tracking branch 'upstream/master' into traitsroundstart

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+//MISC - SKILLS
+/datum/quirk/notools
+	name = "Manos inutiles"
+	desc = "Tienes las mimsas habilidades de construccion que... no sabes constuir nada"
```
Pon: El uso de herramientas se te dificulta.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	mob_trait = TRAIT_NO_TOOLS
+	gain_text = "<span class='danger'>Tus habilidades manuales son basura!</span>"
+	lose_text = "<span class='notice'>Al parecer esto de la mecanica no era tan dificil</span>"
+	medical_record_text = "Al paciente le cuesta realizar habilidades manuales."
```
Deberías borrar todos los medical_record_text si no tienen utilidad alguna.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	value = -1
+	mob_trait = TRAIT_NO_CRAFT
+	gain_text = "<span class='danger'>No podrias ni craftear un barco de papel</span>"
+	lose_text = "<span class='notice'>Esto del maincra no se te daba tan mal.</span>"
```
Pon mejor: Esto de craftear no se te daba tan mal.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+//GENETICA
+/datum/quirk/torpeza
+	name = "Torpeza"
+	desc = "Parece que eres tan torpe como nadie, debes tener algun cromosoma extra"
```
Quita el tan y como nadie.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	name = "Torpeza"
+	desc = "Parece que eres tan torpe como nadie, debes tener algun cromosoma extra"
+	value = -1
+	gain_text = "<span class='danger'>Eres evan, cuidado al caminar!</span>"
```
Cambia evan por torpe.

Evankhell561 committed:
Merge branch 'master' into traitsroundstart

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	desc = "Parece que eres tan torpe como nadie, debes tener algun cromosoma extra"
+	value = -1
+	gain_text = "<span class='danger'>Eres evan, cuidado al caminar!</span>"
+	lose_text = "<span class='notice'>Pues esto de trabajar no era nada del otro mundo.</span>"
```
Sera mejor que pongas: Te sientes habilidoso.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	H.dna.default_blocks.Add(GLOB.clumsyblock)
+	H.check_mutations = TRUE
+
+/datum/quirk/manco
```
Ya te puedes quitar las manos sin este quirk.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	value = 2
+	mob_trait = TRAIT_MECH_PILOT
+	gain_text = "<span class='notice'>Conducir un mecha no ha de ser complicado para ti.</span>"
+	lose_text = "<span class='notice'>Conducir un ripley es como andar en bicicleta decian.</span>"
```
Falta la tilde en decían.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	value = 2
+	mob_trait = TRAIT_POD_PILOT
+	gain_text = "<span class='notice'>Conducir un pod no ha de ser complicado para ti.</span>"
+	lose_text = "<span class='notice'>Conducir un pod es como andar en bicicleta decian.</span>"
```
Lo mismo de la tilde en decían.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	gain_text = "<span class='notice'>Conducir un pod no ha de ser complicado para ti.</span>"
+	lose_text = "<span class='notice'>Conducir un pod es como andar en bicicleta decian.</span>"
+	class = "CONDUCIR"
+	subclass = "POD" //tiene la misma clase  que el mech pilot pero diferente supclase
```
En el comentario pusiste supclase en vez de subclase.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	desc = "Que sea doble, por favor."
+	value = 1
+	gain_text = "<span class='notice'>Has bebido tanto en tu vida que ya el alcohol casi ni te afecta.</span>"
+	lose_text = "<span class='notice'>Ya no tienes tanta resitencia como antes.</span>"
```
Pusiste resitencia en vez de resistencia.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	H.check_mutations = TRUE
+
+/datum/quirk/estomagofuerte
+	name = "Estomago fuerte"
```
Seria mejor impasible.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	value = 1	//vale uno porque es una tonteria
+	mob_trait = TRAIT_ESTOMAGO_FUERTE
+	gain_text = "<span class='notice'>Los olores fuertes no te afectan.</span>"
+	lose_text = "<span class='notice'>Los lores fuertes hacen revolver el estomago.</span>"
```
Pusiste lores en vez de olores.

Commit:
fix

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+
+//MEDBAY SKILLS
+/datum/quirk/cirujano
+	name = "Crujano Veterano"
```
Te falta la i en Cirujano.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+//MEDBAY SKILLS
+/datum/quirk/cirujano
+	name = "Crujano Veterano"
+	desc = "Tienes mucha experiencia haciendo cirugias."
```
Cirguías lleva tilde en la segunda i.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	desc = "Eres mejor tirador a dos armas que cualquiera."
+	value = 1
+	mob_trait = TRAIT_DUAL_SHOOTER
+	gain_text = "<span class='notice'>Por qu� disperar con un arma cuando puedes disparar con dos?.</span>"
```
Te falta la interrogación inicial y disperar se escribe disparar.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	value = 1
+	mob_trait = TRAIT_DUAL_SHOOTER
+	gain_text = "<span class='notice'>Por qu� disperar con un arma cuando puedes disparar con dos?.</span>"
+	lose_text = "<span class='notice'>Al parecer disprar con dos armas no era tan facil despues de todo.</span>"
```
Escribiste mal disparar, fácil lleva tilde y después tambien tiene tilde.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	etiqueta = "SKILL"
+
+/datum/quirk/shooter
+	name = "Tirador PRO"
```
Seria mejor Tirador veterano, al igual que el cirujano.

Silverexperience commented on code/HISPANIA/datums/traits/neutral.dm:
```diff
@@ -1,8 +1,10 @@
 /datum/quirk/no_taste
 	name = "Ageusia"
-	desc = "You can't taste anything! Toxic food will still poison you."
+	desc = "No puedes saborear nada! Las comidas de hace dos semanas te siguen dando indigestion."
```
Te falta la exclamación inicial e indigestión lleva tilde en la o, y no entiendo que tiene que ver el "Las comidas de hace dos semanas te siguen dando indigestion."

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	desc = "Hace mucho perdiste una mano en un accidente."
+	value = -1
+	gain_text = "<span class='danger'>Que le ha pasado a tu mano?</span>"
+	lose_text = "<span class='notice'>Pues parece que al final .</span>"
```
¿Esto esta incompleto?

Silverexperience commented on code/HISPANIA/datums/traits/neutral.dm:
```diff
@@ -1,8 +1,10 @@
 	mob_trait = TRAIT_AGEUSIA
-	gain_text = "<span class='notice'>You can't taste anything!</span>"
-	lose_text = "<span class='notice'>You can taste again!</span>"
+	gain_text = "<span class='notice'>No puedes saborear nada!</span>"
```
Te falta en los dos la exclamación del principio.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+
+/datum/quirk/shooter
+	name = "Tirador PRO"
+	desc = "Est�s capacitado para manejrar armas grandes."
```
Pusiste manejrar en vez de manejar.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	desc = "Est�s capacitado para manejrar armas grandes."
+	value = 1
+	mob_trait = TRAIT_SHOOTER
+	gain_text = "<span class='notice'>Tu punteria aumenta!</span>"
```
Te falta la exclamación inicial y puntería lleva tilde.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	value = 1
+	mob_trait = TRAIT_SHOOTER
+	gain_text = "<span class='notice'>Tu punteria aumenta!</span>"
+	lose_text = "<span class='notice'>Tu punteria ya no es lo que era.</span>"
```
puntería lleva tilde.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	etiqueta = "SKILL"
+
+//ARTES MARCIALES
+/datum/quirk/llamaazul
```
No creo que los quirks tengan que tener artes marciales.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+//IMPLANTES
+/datum/quirk/nutriimplant
+	name = "NutriImplante"
+	desc = "tienes implantado un implante de nutrici�n."
```
Falta la mayuscula inicial.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	name = "NutriImplante"
+	desc = "tienes implantado un implante de nutrici�n."
+	value = 1
+	gain_text = "<span class='notice'>Tienes un implante de nutricion en el cuerpo.</span>"
```
Nutrición lleva tilde en la o.

Silverexperience commented on code/HISPANIA/datums/traits/positive.dm:
```diff
@@ -0,0 +1,119 @@
+	desc = "tienes implantado un implante de nutrici�n."
+	value = 1
+	gain_text = "<span class='notice'>Tienes un implante de nutricion en el cuerpo.</span>"
+	lose_text = "<span class='notice'>Sientes... hambre?.</span>"
```
Falta la interrogación inicial.

Silverexperience commented on code/HISPANIA/game/jobs/job/medical.dm:
```diff
@@ -0,0 +1,58 @@
+		return
+	H.add_quirk(/datum/quirk/cirujano)
+
+//FORENCE - JOB DE ROL
```
Pusiste FORENCE en vez de Forense.

Silverexperience commented on code/HISPANIA/game/jobs/job/medical.dm:
```diff
@@ -0,0 +1,58 @@
+	H.add_quirk(/datum/quirk/cirujano)
+	H.add_quirk(/datum/quirk/estomagofuerte)
+
+//QUIMICO  - JOB DE ROL
```
Químico lleva tilde en la primera i.

Silverexperience commented on code/HISPANIA/game/jobs/job/medical.dm:
```diff
@@ -0,0 +1,58 @@
+		return
+	H.add_quirk(/datum/quirk/cirujano)
+
+//VIROLOGO - JOB DE ROL
```
Virólogo lleva tilde en la primera o.

Silverexperience commented on code/HISPANIA/game/jobs/job/medical.dm:
```diff
@@ -0,0 +1,58 @@
+		return
+	H.add_quirk(/datum/quirk/cirujano)
+
+//SIQUIATRA  - JOB DE ROL
```
Se escribe Psiquiatra.

Silverexperience commented on code/HISPANIA/game/jobs/job/medical.dm:
```diff
@@ -0,0 +1,58 @@
+		return
+	H.add_quirk(/datum/quirk/cirujano)
+
+//PARAMEDICO - JOB DE ESPECIAL
```
Paramédico lleva tilde en la e.

Silverexperience commented on code/HISPANIA/game/jobs/job/medical.dm:
```diff
@@ -0,0 +1,58 @@
+	if(visualsOnly)
+		return
+	H.add_quirk(/datum/quirk/mechpilot)
+	H.add_quirk(/datum/quirk/podpilot) //PARA RESACATES ESPACIALES
```
Pusiste RESACATES en vez de rescates.

Silverexperience commented on code/HISPANIA/game/jobs/job/science.dm:
```diff
@@ -0,0 +1,25 @@
+	H.add_quirk(/datum/quirk/mechpilot)
+	H.add_quirk(/datum/quirk/cirujano)
+
+//CIENTIFICO
```
Científico lleva tilde en la segunda i.

Silverexperience commented on code/game/objects/items/tools/tool_behaviour.dm:
```diff
@@ -1,6 +1,9 @@
 // Handles most checks.
 /obj/item/proc/use_tool(atom/target, mob/living/user, delay, amount=0, volume=0, datum/callback/extra_checks)
+	if(HAS_TRAIT(user, TRAIT_NO_TOOLS) && ishuman(user)) //HISPA TRATIS
+		to_chat(user, "<span class='notice'>no sabrias usar esta herramienta aunque quisieras.</span>")
```
Faltaría la mayúscula inicial y sabrías lleva tilde.

Silverexperience commented on code/modules/mob/living/carbon/human/human.dm:
```diff
@@ -1756,6 +1756,9 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 
 	if(G.trigger_guard == TRIGGER_GUARD_NORMAL)
+		if(HAS_TRAIT(src, TRAIT_GUNS_FOBIA))
+			to_chat(src, "<span class='warning'>Tu fobia a las armas de fuego no te deja apretar el gatillo!</span>")
```
La exclamación inicial.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	etiqueta = "SKILL"
+
+/datum/quirk/gusfobia
+	name = "Guns fobia"
```
Pon mejor: Hoplofobia.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
 	gain_text = "<span class='danger'>You feel repulsed by the thought of violence!</span>"
-	lose_text = "<span class='notice'>You think you can defend yourself again.</span>"
-	medical_record_text = "Patient is unusually pacifistic and cannot bring themselves to cause physical harm."
+	lose_text = "<span class='notice'>Puedes defenderte a ti mismo nuevamente.</span>"
```
A ti mismo seria mejor quitarlo, con defenderte ya se entiende que eres tu al que puedes defender.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
-	lose_text = "<span class='notice'>You think you can defend yourself again.</span>"
-	medical_record_text = "Patient is unusually pacifistic and cannot bring themselves to cause physical harm."
+	lose_text = "<span class='notice'>Puedes defenderte a ti mismo nuevamente.</span>"
+	medical_record_text = "El paciente es inusualmente pacifico y no hará nada que pueda dañar físicamente a otros."
```
Pacífico lleva tilde en la primera i.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	desc = "Le tienes fobia a las armas de fuego."
+	value = -1 //esto vale uno porque si alguien lo toma tambien puede tomar el poor_aim gratis
+	mob_trait = TRAIT_GUNS_FOBIA
+	gain_text = "<span class='danger'>Recuerdas eso... disparos, no de nuevo!</span>"
```
Faltaría la interrogación antes de no.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	value = -1 //esto vale uno porque si alguien lo toma tambien puede tomar el poor_aim gratis
+	mob_trait = TRAIT_GUNS_FOBIA
+	gain_text = "<span class='danger'>Recuerdas eso... disparos, no de nuevo!</span>"
+	lose_text = "<span class='notice'>Pues disparar es hasta divertido, quien lo diria.</span>"
```
Diría lleva tilde en la segunda i.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	desc = "Tienes las mimsas habilidades de construccion que... no sabes constuir nada"
+	value = -1
+	mob_trait = TRAIT_NO_TOOLS
+	gain_text = "<span class='danger'>Tus habilidades manuales son basura!</span>"
```
Faltaría la exclamación inicial.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	value = -1
+	mob_trait = TRAIT_NO_TOOLS
+	gain_text = "<span class='danger'>Tus habilidades manuales son basura!</span>"
+	lose_text = "<span class='notice'>Al parecer esto de la mecanica no era tan dificil</span>"
```
Mecánica y difícil llevan tilde, uno en la primera a y el otro en la segunda i.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	desc = "No puedes craftear nada."
+	value = -1
+	mob_trait = TRAIT_NO_CRAFT
+	gain_text = "<span class='danger'>No podrias ni craftear un barco de papel</span>"
```
Podrías lleva tilde en la i, además falta el punto final.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	etiqueta = "SKILL"
+
+/datum/quirk/nocraft
+	name = "Craftear, que es eso?"
```
Seria mejor que pongas: Desmañado.

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
+	subclass = "CRAFT"
+	etiqueta = "SKILL"
+
+//GENETICA
```
Genética lleva tilde en la segunda e.

Evankhell561 replied:
esto es un comentario de code e.e
lo dejaré asi para que se pueda visualisar bien desde el navegador

Silverexperience commented on code/HISPANIA/datums/traits/negative.dm:
```diff
@@ -1,8 +1,80 @@
 /datum/quirk/nonviolent
-	name = "Pacifist"
+	name = "Pacifista"
 	desc = "The thought of violence makes you sick. So much so, in fact, that you can't hurt anyone."
```
Aún tienes descripciones en ingles.

Evankhell561 replied:
listo

Commit:
fix silver

Commit:
oops

Commit:
opps3

Commit:
OPPS4

Commit:
opps again

Commit:
mardeta sea

Commit:
quirk donas

Commit:
hud quirks

Evankhell561 commented:
Silver, no le puedo poner tiles a los nombres ni descriciones porque causan bugs visuales, tampoco puedo poner ñ. Estas cosas solo se las puedo colocar a los textos ic, que aparecen ic. los que estan marcados como span

Silverexperience reviewed (approved):
Esperemos que funcione.
